src/App.js

```diff
@@ -0,0 +1,201 @@
+import React, { Component } from "react";
+
+import TodoItemsList from "./components/TodoItemsList";
+import TodoInput from "./components/TodoInput";
+import TodoSearch from "./components/TodoSearch";
+import "./App.css";
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "",
+      surname: "",
+      message: "",
+      isSubmitted: false,
+      key: 2,
+      todoItems: [
+        {
+          name: "Andrew",
+          surname: "anime",
+          message: "Hello World!",
+          date: '12.10.2018',
+          key: 1,
+          isCompleted: false,
+          display: true,
+          textMatchStart: "",
+          textMatch: false,
+          textMatchEnd: ""
+        },
+        {
+          name: "Yura",
+          surname: "notanime",
+          message: "Hello World!",
+          date: '12.10.2018',
+          key: 2,
+          isCompleted: true,
+          display: true,
+          textMatchStart: "",
+          textMatch: false,
+          textMatchEnd: ""
+        }
+      ]
+    };
+  }
+
+  handleChange = e => {
+    this.setState({ [e.target.name]: e.target.value })
+  };
+
+  handleAdd = e => {
+    let newKey = this.state.key;
+    newKey++;
+    this.setState({ isSubmitted: true, key: newKey });
```

Comment:
`isSubmitted` where is it used?

Reply:
It's created to emulate ajax request later.